package.json

```
@@ -32,7 +32,7 @@
     "react-moment": "0.9.2",
     "react-redux": "5.1.1",
     "react-router-dom": "5.0.1",
-    "react-scripts": "3.0.1",
+    "react-scripts": "^3.1.1",
```

Version should be fixed.

```suggestion
"react-scripts": "3.1.1",
```